src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -1686,4 +1686,11 @@ All WinRM sessions connected to PowerShell session configurations, such as Micro
   <data name="UnixOnlyHttpsWithoutSkipCACheckNotSupported" xml:space="preserve">
     <value>HTTPS on Unix does not currently support CA or CN checks. Use the PSSessionOption -SkipCACheck and -SkipCNCheck if you are certain you trust the server you are connecting to and the network in between.</value>
   </data>
-</root>
+  <data name="PSCoreRemotingDisableWarning" xml:space="preserve">
+    <value>PowerShell remoting has been disabled only for PowerShell Core configurations and does not affect Windows PowerShell remoting configurations. Run this cmdlet in Windows PowerShell to affect all PowerShell remoting configurations.
+</value>
+  </data>
+  <data name="PSCoreRemotingEnableWarning" xml:space="preserve">
+    <value>PowerShell remoting has been enabled only for PowerShell Core configurations and does not affect Windows PowerShell remoting configurations. Run this cmdlet in Windows PowerShell to affect all PowerShell remoting configurations.</value>
+  </data>
+</root>
```

Comment:
missing newline at end of file

Reply:
This is intended.  No newline is needed in this case.